packages/realm_dart/test/dynamic_realm_test.dart

```diff
@@ -438,24 +646,24 @@ void main() {
 
         final obj = dynamicRealm.dynamic.all(AllCollections.schema.name).single;
         expect(
-            () => obj.dynamic.get<String>('strings'),
+            () => obj.dynamic.get<String>('stringList'),
             throws<RealmException>(
-                "Property 'strings' on class 'AllCollections' is 'RealmCollectionType.list' but the method used to access it expected 'RealmCollectionType.none'."));
+                "Property 'stringList' on class 'AllCollections' is 'RealmCollectionType.list' but the method used to access it expected 'RealmCollectionType.none'."));
```

Comment:
I hope we can give a better error message at some point, instead of leaking the internal representation stuff.